presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -772,6 +773,19 @@ public HiveClientConfig setPinS3ClientToCurrentRegion(boolean pinS3ClientToCurre
         return this;
     }
 
+    public String getS3UserAgentPrefix()
```

Comment:
Side note: if we were to keep the default as an empty string, this should be validated as `@NotNull`.